<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_sinfo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>SF_INFO</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_stats">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Stats</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainText_sinfo">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>341</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_stats">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>40</y>
      <width>391</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_play">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_filename">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>270</y>
      <width>581</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>filename</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="customPlot" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>741</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_zoom">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>510</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Zoom</string>
    </property>
   </widget>
   <widget class="QScrollBar" name="horizontalScrollBar_zoom">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>510</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave_as"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open..</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
